contrib/depends/patches/native_libtapi/no_embed_git_rev.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/src/llvm/CMakeLists.txt b/src/llvm/CMakeLists.txt
+index ab92717c8..dff3a292a 100644
+--- a/src/llvm/CMakeLists.txt
++++ b/src/llvm/CMakeLists.txt
+@@ -754,7 +754,6 @@ set(LLVM_SRPM_DIR "${CMAKE_CURRENT_BINARY_DIR}/srpm")
+ 
+ # SVN_REVISION and GIT_COMMIT get set by the call to add_version_info_from_vcs.
+ # DUMMY_VAR contains a version string which we don't care about.
```

Comment:
Since you're deleting the call to add_version_info_from_vcs() it seems the above comments are no longer relevant or accurate. Is there anything you can add here that makes it clear what's really happening now?

Reply:
I updated the comment.